### PR TITLE
Vbull/triangle

### PR DESCRIPTION
**Pull request by Manbearpixel**
Went back and adjusted styling, re-formated some code

### CODE REVIEW

File: triangle_facts/triangle_facts.rb [hunk 1]
```diff
@@ -1,40 +1,44 @@
 # Killer facts about triangles AWW YEAH
 class Triangle
-	attr_accessor :side1,:side2,:side3
 
-	def initialize(side1,side2,side3)
-		@side1,@side2,@side3 = side1,side2,side3
+	attr_accessor :side1, :side2, :side3
+	attr_reader :angles
+
+	def initialize(side1, side2, side3)
```
Comment (johnmcconnell) [inline triangle_facts/triangle_facts.rb]: makes hash initializing easier. https://github.com/johnmcconnell/attr_init
Also, look into supporting default and named parameters
```ruby
def add(_x = nil, _y = nil, x: _x, y: _y)
  x + y
end
add(2, 3) # => 5
add(x: 2, y: 3) # => 5 
```

File: triangle_facts/triangle_facts.rb [hunk 2]
```diff
@@ -1,40 +1,44 @@
...
 
-	def equilateral()
+	def equilateral?
 		return side1 == side2 && side2 == side3
 	end
 
-	def isosceles()
-		return [side1,side2,side3].uniq.length == 2
+	def isosceles?
+  	return [side1, side2, side3].uniq.length == 2
```
Comment (johnmcconnell) [inline triangle_facts/triangle_facts.rb]: fix spacing and remove the return
```ruby
def isosceles?
  [side1, side2, side3].uniq.length == 2
end
```

File: triangle_facts/triangle_facts.rb [hunk 3]
```diff
@@ -1,40 +1,44 @@
...
 	end
 
-	def scalene()
-		unless ! (equilateral || isosceles)
-	    false
-	  else
-	    true
-	  end
+	def scalene?
+		equilateral? || isosceles?
```
Comment (johnmcconnell) [inline triangle_facts/triangle_facts.rb]: ahhh, isn't it
```ruby
!(equilateral? || isoceles?)
```

File: triangle_facts/triangle_facts.rb [hunk 4]
```diff
@@ -1,40 +1,44 @@
...
+		angles.include? 90
+	end
 
-		puts 'This triangle is also a right triangle!' if angles.include? 90
-		puts ''
+	def recite_facts
+		triangleFacts = ""
+		triangleFacts = "This triangle is equilateral!" if equilateral?
+		triangleFacts = "This triangle is isosceles! Also, that word is hard to type." if isosceles?
+		triangleFacts = "This triangle is scalene and mathematically boring." unless scalene?
```
Comment (johnmcconnell) [inline triangle_facts/triangle_facts.rb]: this line is an example of the backwards logic. tell me its scalene unless it is scalene

File: triangle_facts/triangle_facts.rb [hunk 5]
```diff
@@ -1,40 +1,44 @@
...
-		angles = self.calculate_angles(side1,side2,side3)
-		puts 'The angles of this triangle are ' + angles.join(',')
+	def right_angle?
+		angles.include? 90
+	end
 
-		puts 'This triangle is also a right triangle!' if angles.include? 90
-		puts ''
+	def recite_facts
+		triangleFacts = ""
```
Comment (johnmcconnell) [inline triangle_facts/triangle_facts.rb]: this should be useless because a triangle must be on of equilateral, isosceles, or scalene

**Comment by paulghaddad** [comment]
@Manbearpixel Please take a look at the comments and resubmit.

**Pull request closed without merging**